docs/guides/admin/docs/workflowoperationhandlers/concat-woh.md

```diff
@@ -24,19 +36,57 @@ FFmpeg Filter Example
       aevalsrc=0::d=1[silent];
       [b][0:a][c][silent][d][2:a]concat=n=3:v=1:a=1[v][a]' -map '[v]' -map '[a]'
 
+
+
+Same Codec Mode
+---------------
+
+*Requires the source tracks having the same format (same timebase/resolution/encoding, etc.)*
+
+If the `same-codec` option is specified to use this mode, the sources files can be arranged into one container
+losslessly without re-encoding first.  This is often the case if the tracks came from the same camera/recorder for
+example.
+
+This mode uses [FFmpeg's concat demuxer](https://www.ffmpeg.org/ffmpeg-formats.html#concat-1), which puts all the video
+content into a single container without any re-encoding. The encoding profile then operates on the source in this
+container. If `-c copy` is used in the encoding profile, the complete concatenation is lossless.
+
+The FFmpeg command for this is is:
+
+    -f concat -i videolist.txt
+
+…where `videolist.txt` contains a line in the form `file <path to video>` for each source track.
+
+
 Usage
 -----
 
 This operation is quite similar to the compose operation. The only difference is that the input properties are not only
```

Comment:
Are not only limited?  Or are limited?  The latter is how the rest of the sentence reads.

Reply:
Old docs :-) 
But I also think this is fine as it is.